### DISCUSSION TITLE
Changing charge limit and charging amps fails

### DISCUSSION THREAD

**Discussion by tnolf** [alandtse/tesla#800] [General]
Hi,

does someone else get an unknown error when changing the charge limit or charging amps? 

![IMG_5519](https://github.com/alandtse/tesla/assets/1489770/210dfa80-e2f0-4e30-a4d1-435ddf2dd4b0)

Best
Tom

**Comment by tnolf** [comment]
Figured it out. It’s because Tesla is shutting down restful commands in their API and replacing them with a command SDK. New cars from November 2023 on are on the new SDK only already whereas older cars will see the restful commands been pulled away from January 2024. 
So more people will have the same issue as I have in a couple of weeks I am afraid.

**Reply by IanK6449** [reply]
I have the same problem with a new Model Y. But, does the force_update button need to be 'pressed' in order for any such updates to work? I too am trying to set the charging amps in order to optimise use of surplus solar power but I get the same error.
Any ideas whether this is something that can be fixed in the integration or is Tesla trying to shut down such integrations with their API changes?

**Reply by skipishere** [reply]
See issue #743
